src/bin/pallene-debug

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env lua
+
+if #arg == 0 then
+    io.stderr:write("pallene-debug: need a file to debug. Abort!\n")
+    os.exit(1)
```

Comment:
Consider using argparse. Gives us --help for free, standard "usage" message, etc. See `pallenec` main script.

Reply:
Done.